src/node.rs

```diff
@@ -238,8 +246,8 @@ impl InMemoryNode {
         let mut inner = self.inner.write().unwrap();
         let keys = {
             let mut storage_view = StorageView::new(&inner.fork_storage);
-            storage_view.set_value(&key, u256_to_h256(U256::from(10u128.pow(22))));
-            storage_view.get_modified_storage_keys().clone()
+            storage_view.set_value(key, u256_to_h256(U256::from(10u128.pow(19))));
```

Comment:
Did you intend to reduce the amount of ETH we give Rich accounts?

Reply:
Nope will update to reflect the previous change.

Reply:
@MexicanAce updated.